crates/ironrdp-cliprdr-format/src/html.rs

```diff
@@ -60,7 +60,7 @@ pub fn cf_html_to_plain_html(input: &[u8]) -> Result<&str, HtmlError> {
                     let end = usize::try_from(end).map_err(|_| HtmlError::InvalidConversion)?;
 
                     // Ensure start and end values are properly bounded.
-                    if start > end || end > input.len() {
+                    if !(start < end && end < input.len()) {
```

Comment:
Apply style suggested by "State invariants positively" and "Strongly prefer < and <= over > and >=", which I think is easier to glimpse at